src/editor/SettingsDialog.cpp

```diff
@@ -11,7 +15,31 @@ SettingsDialog::SettingsDialog(const MidiWrapper *midi_wrapper, QWidget *parent)
       ui(new Ui::SettingsDialog) {
   ui->setupUi(this);
 
+  const QStringList styles = QStyleFactory::keys();
+
+  int styleIterator = 0;
+  int styleIndex = 0;
+  styleFile.open(QFile::ReadOnly);
```

Comment:
Is there a particular reason the setting's in a custom file over something in QSettings? (An example QSettings use is Settings::CustomStyle which you've seen / used below. The Settings namespace is maybe a bit of a weird usage of QSettings but it's the pattern in this project at least)

Reply:
Yes. The style information is kept in a distinctive file rather than a QSettings key because some styles have a tendency to crash the program. In the event of a crash, you would not be unable to change any QSettings, whereas the modification/deletion of the .style file will fix the problem entirely.

Reply:
Oh, I see what you mean now regarding the namespace. No idea why I didn't just put it externally or something. It should work the same